content/pt/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+ - femrtnz
+ - jcjesus
+title: Ambiente de Contêiner
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+Essa página descreve os recursos disponíveis para contêineres no ambiente de contêiner.
+
+
+
+<!-- body -->
+
+## Ambiente de contêiner
+
+O ambiente de contêiner do Kubernetes fornece recursos importantes para contêineres: 
+
+* Um sistema de arquivos, que é a combinação de uma [imagem](/docs/concepts/containers/images/) e um ou mais [volumes](/docs/concepts/storage/volumes/).
+* Informação sobre o contêiner propriamente.
+* Informação sobre outros objetos no cluster.
+
+### Informação de contêiner
+
+O _hostname_ de um contêiner é o nome do Pod em que o contêiner está executando.
+Isso é disponibilizado através do comando `hostname` ou da função [`gethostname`](https://man7.org/linux/man-pages/man2/gethostname.2.html) chamada na libc.
+
+O nome do Pod e o Namespace são disponibilizados como variáveis de ambiente através do [downward API](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).
```

Comment:
esse downward API aqui, será que não tem alguma palavra melhor? Eu to até agora tentando entender :D

Reply:
As traduções literais seriam algo para baixo ou descendente haha, poderíamos reescrever como:
"...são expostos como variáveis de ambiente através de um mecanismo chamado [Downward API]"

Ou algo assim, o mais parecido com definição seria:
["The Downward API allows containers to consume information about themselves or the cluster without using the Kubernetes client or API server. "](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#motivation-for-the-downward-api)

Reply:
Gostei :D